packages/code-du-travail-data/dataset/editorial_content/index.js

```diff
@@ -0,0 +1,79 @@
+import { getRouteBySource, SOURCES } from "@cdt/sources";
+import filesize from "filesize";
```

Comment:
using a package to get the filesize quite surprises me !

Reply:
it allows to not have to do the logic of o / Ko / Mo / b /Kb /Mb Gb, rounding and co